proposals/fh_discord.md

```diff
@@ -0,0 +1,47 @@
+# Florida Hackers Discord Chat 
+
+## Target audience
+
+<!-- Who are you expecting to impact based on your proposal? -->
+Not only will the discord improve communication for FH Organizers with permission based rooms(The ability to create rooms made for specific eyes only based on roles assigned in discord, which an infinite amount of roles can be assigned to, where rooms can be seen by everyone or a select few depending on assigned role), it also allows a more open community of Florida Hackers to be active on discord from the rest of Florida.
+
+## Community needs
+
+<!-- What specific issue are you addressing with your proposal? -->
+When it comes to determing whether or not Discord should be used over Slack, it's important to note that Slack isn't entirely free and has a data limit for messages and files for the free tier(That info can be found here https://get.slack.help/hc/en-us/articles/115002422943-Message-and-storage-limits-on-the-Free-plan). 
+
+With Discord, Florida Hackers will have the ability to grow its community without the need for limiting the amount of users in the chat. 
+
+Currently the Florida Hackers Slack channel is not as active as it used to be, and we have already surpassed our 10,000 message free limit meaning that older messages are getting deleted over time. We only have 1.2 GB left of file storage left. The last message made on our general chat was on the day May 30th.As of Wednesday, August 29th we only hit 2 daily active users with the all time high being attributed to August 22nd with 3 daily active users. 
+
+Surely the Florida Hackers community is more than just 2-3 daily active users, therefore I find it vital that we swtich to discord as a platform in order to grow a successful community! 
```

Comment:
Is there any sort of data to back up that moving from Slack to Discord will actually make the community more active?

Reply:
Wanna follow up on this. @carloslahrssen I/you can set up a poll to start validating this idea.